dummyserver/server.py

```diff
@@ -16,6 +16,11 @@
 import warnings
 from datetime import datetime
 
+try:
+    from ssl import CERT_OPTIONAL
+except:
+    CERT_OPTIONAL = None
```

Comment:
I'm fine with the blind import. Realistically a Python without an `ssl` module is next to useless for HTTP, so I don't really mind.